src/main/java/com/moneytransfer/entity/Request.java

```diff
@@ -0,0 +1,70 @@
+package com.moneytransfer.entity;
+
+import com.moneytransfer.dto.NewTransferDto;
+import com.moneytransfer.enums.RequestStatus;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ *  Entity that represents an idempotent transfer request.
+ */
+@Entity
+@Table(name = "requests")
+@Getter
+@NoArgsConstructor(access = lombok.AccessLevel.PRIVATE)
+public class Request implements Serializable {
```

Comment:
I guess something like CreateTransactionRequest or something like this is more descriptive.

Reply:
done!